### ISSUE TITLE
SmartCity-Accessibility #403 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#244]
Build 'SmartCity-Accessibility' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] d167ee0b366a01d8ef8f5cc3784e478d43c788c0 - add client side(Ionic) and split project

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/403/)

**Comment by kapalex** [comment]
@yaelAmitay I'll take care of the build issues in jenkins, but it seems like theres no files in the repo right now

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] d167ee0b366a01d8ef8f5cc3784e478d43c788c0 - add client side(Ionic) and split project

- [yaelAmitay] 21da45720832ca28d99f98d928360a84df155972 - add client side(Ionic) and split project



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/404/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] d167ee0b366a01d8ef8f5cc3784e478d43c788c0 - add client side(Ionic) and split project

- [yaelAmitay] 21da45720832ca28d99f98d928360a84df155972 - add client side(Ionic) and split project



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/405/)

**Comment by yaelAmitay** [comment]
yes i forgot to do "git add ." lol
but now it's suppose to be fine

**Comment by JuanJenkinsBot** [comment]
Build was fixed!

**Comment by kapalex** [comment]
for people pulling the project after the change you can use `git clean -fd` to remove files left over by the previous structure @koralchapnik @Kolikant @ArthurSap
